### DISCUSSION TITLE
What do you think?

### DISCUSSION THREAD

**Discussion by xames3** [ciesin-geospatial/TOPSTSCHOOL-OSDMP#44] [Ideas & Feedbacks]
Tell us what you think about the content so far!

**Comment by jfmartinez4** [comment]
Looking good!

**Reply by xames3** [reply]
We can record all the feedbacks here. If needed, we can have [Polls](https://github.com/ciesin-geospatial/TOPSTSCHOOL-OSDMP/discussions/categories/polls) and [QnA](https://github.com/ciesin-geospatial/TOPSTSCHOOL-OSDMP/discussions/categories/q-a)'s as well in different sections of the discussions.
